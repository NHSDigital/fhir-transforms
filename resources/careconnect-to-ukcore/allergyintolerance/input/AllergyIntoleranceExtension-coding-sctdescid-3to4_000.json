{
  "resourceType": "AllergyIntolerance",
  "code": {
    "coding": [
      {
        "system": "http://read.info/readv2",
        "code": "TJ03000",
        "display": "Adverse reaction to erythromycin",
        "userSelected": true
      },
      {
        "extension": [
          {
            "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid",
            "extension": [
              {
                "url": "descriptionId",
                "valueId": "2579700012"
              },
              {
                "url": "descriptionDisplay",
                "valueString": "Adverse reaction to erythromycin"
              }
            ]
          }
        ],
        "system": "http://snomed.info/sct",
        "code": "419056004",
        "display": "Erythromycin adverse reaction"
      }
    ]
  }
}
