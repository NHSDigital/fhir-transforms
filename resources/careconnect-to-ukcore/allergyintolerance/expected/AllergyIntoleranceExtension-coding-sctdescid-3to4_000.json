{
  "resourceType": "AllergyIntolerance",
  "code": {
    "coding": [
      {
        "extension": [
          {
            "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-CodingSCTDescId",
            "extension": [
              {
                "url": "descriptionId",
                 "valueIdentifier": {
                   "system": "http://snomed.info/sct",
                   "value": "152307010"
                 }
              },
              {
                "url": "descriptionDisplay",
                "valueString": "Allergy to penicillin (finding)"
              }
            ]
          }
        ],
        "system": "http://snomed.info/sct",
        "code": "91936005",
        "display": "Allergy to penicillin"
      }
    ],
    "text": "Penicillin allergy"
  }
}
